### ISSUE TITLE
Filenames with square brackets ignored

### ISSUE THREAD

**Issue by kwbr** [tfeldmann/organize#348]
**Describe the bug**
When using the extension filter, filenames with square brackets are not correctly echoed:

```
$ ls Downloads/*.opus
'Downloads/hello [world].opus'
$ organize run test.yaml
Config: "test.yaml"

⚙ Rule #0: Upload mixes ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
/home/kai/Downloads
  ./hello .opus
    - (echo) /home/kai/Downloads/hello .opus

success 1 / fail 0
```

The path is correctly used for example in `shell` actions but the output is misleading.

Interestingly these test passes:

```python
def test_echo_path_with_spaces(fs, testoutput):
    make_files(["file A.txt", "file B with [brackets].txt"], "test")
    Config.from_string(
        """
        rules:
          - locations: /test
            actions:
              - echo: "{path}"
        """
    ).execute(simulate=False, output=testoutput)
    assert Counter(testoutput.messages) == Counter(["/test/file A.txt", "/test/file B with [brackets].txt"])
    
def test_filename_with_spaces(fs, testoutput):
    files = {
        "test file with spaces.jpg": "",
        "test file with [brackets].jpg": "",
    }
    make_files(files, "test")
    config = """
        test_ext: &test
            - jpg
        rules:
        - locations: /test
          filters:
            - extension: *test
          actions:
            - echo: 'Found file: {path}'
        """
    Config.from_string(config).execute(simulate=False, output=testoutput)
    assert testoutput.messages == [
        "Found file: /test/test file with spaces.jpg",
        "Found file: /test/test file with [brackets].jpg"
    ]

```

**Environment (please complete the following information):**

- OS: Linux, pipx installed organize
- Output of `organize --version`: organize v3.0.1

**Your config file**

```yaml
audio_ext: &audio
  - m4a
  - mp3
  - wav
  - ogg
  - opus

rules:

  - name: "Upload mixes"
    locations:
      - ~/Downloads
    filters:
      - extension: *audio
    actions:
      - echo: "{path}"
```

**Comment by tfeldmann** [comment]
Thanks for reporting! Organize uses the `rich` library for the console output, which uses brackets for styling (`[red]Some text[/]`) and I totally forgot to escape file paths and messages.

The tests still work because we use `output.saving.SavingOutput` in the tests, which has no concept of styling.

This will be fixed in the next release!

**Comment by tfeldmann** [comment]
Fixed in [530c433](https://github.com/tfeldmann/organize/commit/530c433190915b178c3e860ea231155d63198947)
